modules/dataquery/jsx/react.importCSV.js

```diff
@@ -0,0 +1,136 @@
+/**
+ *  The following file handles importing CSV file of either PSCID or DCCID identifiers.
+ *  Used for populating the filter of the DQT and to filter out the demographic from the identifiers.
+ *
+ *  @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ *  @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ *  @link     https://github.com/mohadesz/Loris-Trunk
+ */
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Modal from 'jsx/Modal';
+import Papa from 'papaparse';
+
+/*
+ *  The following component displays a modal
+ *  for extracting PSCID or DCCID identifiers from a CSV file.
+ */
+class ModalImportCSV extends Component {
```

Comment:
This modal window did not display the file name I chose after I selected a file to import. Keep in mind though that I tested this on a refactored/adapted version of your PR for IBIS so if you cannot reproduce on your system your should probably dismiss this comment.